install/gcp/deployment_manager/istio-cluster.jinja

```diff
@@ -110,14 +116,14 @@ resources:
                 --service istio-sidecar-injector \
                     --namespace istio-system \
                     --secret sidecar-injector-certs
-            kubectl apply -f install/kubernetes/istio-sidecar-injector-configmap-release.yaml
+            $KUBECTL apply -f install/kubernetes/istio-sidecar-injector-configmap-release.yaml
             cat install/kubernetes/istio-sidecar-injector.yaml | \
                 ./install/kubernetes/webhook-patch-ca-bundle.sh > \
                 install/kubernetes/istio-sidecar-injector-with-ca-bundle.yaml
-            kubectl apply -f install/kubernetes/istio-sidecar-injector-with-ca-bundle.yaml
+            $KUBECTL apply -f install/kubernetes/istio-sidecar-injector-with-ca-bundle.yaml
```

Comment:
webhook-patch-ca-bundle.sh no longer needed, since we don't need to manually generate cert and patch it now. Currently, we only need to generate the sidecar yaml from Helm and then call kubectl apply directly.

Reply:
For which version is that true? 0.7.0? 

Note that this patch applies to 0.6.0 and 0.5.0 right now; the template needs to be bumped up to use the latest version (which will happen likely after this CL; I'm just focusing on getting it to a working state before we update the version).

Reply:
The webhook script is removed from the head version and will no longer exist in the upcoming 0.8. It is fine if you are only targeting older versions. 

For deployment manager script, is it always referring some released version? E.g, to support 0.8, do we need to update the script before or after 0.8 is released?

Reply:
It has a hardcoded list of available versions, and always downloads the published release for the selected version. In general we allow the template to drag behind a bit from the current release.